### DISCUSSION TITLE
Is it possible to be notified when there is a change to a schema in the registry?

### DISCUSSION THREAD

**Discussion by chris-giblin** [Apicurio/apicurio-registry#3516] [General]
Have there ever been discussions around use of webhooks, web sockets, etc. in order for clients to be notified when a schema in the registry changes?
Thanks, -chris

**Comment by andreaTP** [comment]
Hi @chris-giblin we have a configurable CloudEvent event-sourcing functionality that is [documented here](https://www.apicur.io/registry/docs/apicurio-registry/2.4.x/getting-started/assembly-configuring-the-registry.html#registry-cloud-events_registry).

Beware that we found a bug ( #3482 ), so you might need to wait for the fix to be released to test it.

**Reply by EricWittmann** [reply]
This is the way.

Also note that we would probably classify this feature as experimental. We are hoping to give it a little more attention for version 3.
